### DISCUSSION TITLE
Discord Server ?

### DISCUSSION THREAD

**Discussion by Xman1109** [Xman1109/Atomic_Overhaul#24] [Polls]
Should I create a discord server for the discussions?
I guess people are more active on Discord than on GitHub.

So it would probably make more sense to create an own discord server for all my factorio mods like [Earendel](https://discord.gg/ymjUVMv) did, right?

Let me know via this poll!

**Comment by Spotticus10290203** [comment]
So did you ever make the server?

**Reply by Xman1109** [reply]
At the moment, no
I think there won't be many people that would join the server. BUT since you reminded me of it, I'll create a server soon.

**Reply by Xman1109** [reply]
https://discord.gg/5mUQUDBcnm
